dali/base/dasds.cpp

```diff
@@ -5735,6 +5735,24 @@ IStoreHelper *createStoreHelper(const char *storeName, const char *location, con
 
 ///////////////
 
+static CConfigUpdateHook configUpdateHook;
+static void initializeStorageGroups(IPropertyTree *oldEnvironment) // oldEnvironment always null in containerized
+{
+    bool forceGroupUpdate = getComponentConfigSP()->getPropBool("DFS/@forceGroupUpdate");
+    initClusterAndStoragePlaneGroups(forceGroupUpdate, oldEnvironment);
+    if (isContainerized())
+    {
+        auto updateFunc = [](const IPropertyTree *oldComponentConfiguration, const IPropertyTree *oldGlobalConfiguration)
+        {
+            bool forceGroupUpdate = getComponentConfigSP()->getPropBool("DFS/@forceGroupUpdate");
+            initClusterAndStoragePlaneGroups(forceGroupUpdate, nullptr);
+        };
+        configUpdateHook.installOnce(updateFunc, true);
```

Comment:
May be harmless, but updateFunc will be called twice - once on line 5742 and once from here

Reply:
unintentional, will change.

Reply:
have changed to:
``
        configUpdateHook.installOnce(updateFunc, false);
```

and squashed.

Reply:
@ghalliday - please see squashed in change.